portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,72 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Nnenna's Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+ <body>
+
+     
+    <h1>Nnenna Dara</h1>
+
+    <div class="tab">
+        <button class="tablinks" onclick="openCity(event, 'About Me')" id="defaultOpen">About Me</button>
```

Comment:
This method doesn't look like it deals with any actual cities. Please rename more appropriately - say openTab or highlightTab?